.github/CODEOWNERS

```diff
@@ -4,71 +4,11 @@
-*                       @googleapis/yoshi-java @googleapis/googleapis-auth
-
-# for handwritten libraries, keep codeowner_team in .repo-metadata.json as owner
-**/*.java               @googleapis/googleapis-auth
-
+# The @googleapis/cloud-java-team-teamsync is the default owner for changes in this repo
+*                       @googleapis/cloud-java-team-teamsync @googleapis/googleapis-auth
 
 # The java-samples-reviewers team is the default owner for samples changes
 samples/**/*.java       @googleapis/java-samples-reviewers
 
 # Generated snippets should not be owned by samples reviewers
-samples/snippets/generated/       @googleapis/yoshi-java
-
-# 3PI-related files and related base classes - joint ownership between googleapis-auth and aion-sdk
-oauth2_http/java/com/google/auth/oauth2/ActingParty.java                      		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java                   		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsSecurityCredentialsSupplier.java   		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsDates.java                         		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsRequestSignature.java              		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsRequestSigner.java                 		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/AwsSecurityCredentials.java           		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/CredentialAccessBoundary.java         		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/DefaultCredentialsProvider.java       		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/DownscopedCredentials.java            		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/EnvironmentProvider.java              		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/ExecutableHandler.java                		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/ExecutableResponse.java               		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java       		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/FileIdentityPoolSubjectTokenSupplier.java 	@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java                		@googleapis/googleapis-auth @googleapis/aion-sdk
-oauth2_http/java/com/google/auth/oauth2/IdentityPoolCredentials.java          		@googleapis/googleapis-auth @googleapis/aion-sdk
```

Comment:
@arithmetic1728 I'm not sure if these rules still apply. If you think these should still be kept, I can revert this section and add cloud java as owners here as well.

Reply:
Ok sounds good. I have reverted the change and added cloud java as additional owners.